locales/en.json

```diff
@@ -1,15 +1,15 @@
 {
 	"Collaborative markdown notes": "Collaborative markdown notes",
-	"Realtime collaborative markdown notes on all platforms.": "Realtime collaborative markdown notes on all platforms.",
-	"Best way to write and share your knowledge in markdown.": "Best way to write and share your knowledge in markdown.",
+	"Realtime collaborative markdown notes on all platforms.": "Callaborate on markdown notes on all platforms.",
+	"Best way to write and share your knowledge in markdown.": "The best way to write and share your knowledge in markdown.",
```

Comment:
Omitting the logo generally does not mean being able to omit the "the". I've seen it many times and it always bothered me, the missing "the".